src/SUMMARY.md

```diff
@@ -1,14 +1,15 @@
 * [Firefox Data Documentation](introduction.md)
-  * [Reporting a problem](concepts/reporting_a_problem.md)
-  * [Terminology](concepts/terminology.md)
+* [Standard Metrics](metrics/index.md)
 * [Getting Started](concepts/getting_started.md)
+  * [Reporting a problem](concepts/reporting_a_problem.md)
   * [Gaining Access](concepts/gaining_access.md)
   * [Analysis Quick Start](concepts/analysis_intro.md)
   * [Choosing a Desktop Dataset](concepts/choosing_a_dataset.md)
   * [Choosing a Mobile Dataset](concepts/choosing_a_dataset_mobile.md)
   * [Intro to STMO](tools/stmo.md)
   * [Common Analysis Gotchas](concepts/analysis_gotchas.md)
   * [Optimizing Queries](concepts/sql_optimization.md)
+  * [Terminology](concepts/terminology.md)
   * [Getting Help](concepts/getting_help.md)
```

Comment:
A few suggestions on ordering. 

* Keep *Standard Metrics* as part of *Getting Started*. 
* Consider merging *Reporting a problem* into the *Meta* or *Getting Help* section
* General reordering for flow
 

```suggestion
* [Getting Started](concepts/getting_started.md)
  * [Gaining Access](concepts/gaining_access.md)
  * [Analysis Quick Start](concepts/analysis_intro.md)
  * [Standard Metrics](metrics/index.md)
  * [Common Analysis Gotchas](concepts/analysis_gotchas.md)
  * [Choosing a Desktop Dataset](concepts/choosing_a_dataset.md)
  * [Choosing a Mobile Dataset](concepts/choosing_a_dataset_mobile.md)
  * [Intro to STMO](tools/stmo.md)
  * [Optimizing Queries](concepts/sql_optimization.md)
  * [Terminology](concepts/terminology.md)
  * [Getting Help](concepts/getting_help.md)
  * [Reporting a problem](concepts/reporting_a_problem.md)
```